pom.xml

```diff
@@ -370,6 +354,12 @@
                     <generateBackupPoms>false</generateBackupPoms>
                 </configuration>
             </plugin>
+            <!-- https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-dependency-plugin -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
I am not 100% sure what we achieve by adding this dependency. As far as I can see, nothing. But I'd be curious to know.

Reply:
I remember using this plugin to look at dependencies and debug. I found it useful, I think I took it from here:

https://www.baeldung.com/maven-dependency-graph

If you want, we can move it to a sort of "debug" dependency